src/RatingAPILayer.js

```diff
@@ -14,10 +14,11 @@ class RatingAPILayer extends React.PureComponent {
     this.handleHover = this.handleHover.bind(this);
   }
 
-  UNSAFE_componentWillReceiveProps(nextProps) {
-    this.setState({
-      value: nextProps.initialRating
-    });
+  componentWillUpdate(prevProps) {
```

Comment:
Is not `componentWillUpdate` deprecated? Did you mean `componentDidUpdate`?

Reply:
:man_facepalming: yes, you're right. I'll tweak that.